From a79fed1bece2e33759546f97ce0f22af3745ac80 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Mon, 21 Dec 2009 13:03:03 -0500
Subject: [PATCH] Allow debug_timestamps setting on a per-domain basis

This was missing from the SSSDConfig API, though it was supported
by the daemon.
---
 server/config/SSSDConfigTest.py |    2 ++
 server/config/etc/sssd.api.conf |    1 +
 2 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/server/config/SSSDConfigTest.py b/server/config/SSSDConfigTest.py
index f893d98..1e79342 100755
--- a/server/config/SSSDConfigTest.py
+++ b/server/config/SSSDConfigTest.py
@@ -372,6 +372,7 @@ class SSSDConfigTestSSSDDomain(unittest.TestCase):
         options = domain.list_options()
         control_list = [
             'debug_level',
+            'debug_timestamps',
             'min_id',
             'max_id',
             'timeout',
@@ -586,6 +587,7 @@ class SSSDConfigTestSSSDDomain(unittest.TestCase):
         options = domain.list_options()
         control_list = [
             'debug_level',
+            'debug_timestamps',
             'min_id',
             'max_id',
             'timeout',
diff --git a/server/config/etc/sssd.api.conf b/server/config/etc/sssd.api.conf
index c8ddae4..bdb6aab 100644
--- a/server/config/etc/sssd.api.conf
+++ b/server/config/etc/sssd.api.conf
@@ -42,6 +42,7 @@ chpass_provider = str, None
 [domain]
 # Options available to all domains
 debug_level = int, None, 0
+debug_timestamps = bool, None
 command = str, None
 min_id = int, None, 1000
 max_id = int, None
-- 
1.6.5.2

